interfaces/cython/cantera/examples/reactors/pfr.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+This example solves a plug-flow reactor problem of hydrogen-oxygen combustion.
+The PFR is computed by two approaches: The simulation of a Lagrangeian fluid
+particle, and the simulation of a chain of reactors.
+"""
```

Comment:
I'd be curious to see a timing difference between these two methods included. I'm assuming that the Lagrangian approach is much faster.